src/app/my-blog/components/blog-item/blog-item.component.html

```diff
@@ -0,0 +1,34 @@
+<li>
```

Comment:
bad HTML semantic, do not use list items

Reply:
fixed